docs/topics/mtr-rn-known-issues-1-2-3.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * docs/release-notes-mtr/master.adoc
+
+:_content-type: REFERENCE
+[id="mtr-rn-known-issues-1-2-3_{context}"]
+
+= Known issues
+
+For a complete list of all known issues, see the list of link:https://issues.redhat.com/issues/?filter=12419394#[MTR 1.2.3 known issues] in Jira.
```

Comment:
The filter is pointing to MTR 1.2.0 Known Issues

Reply:
Updated filter link.